canonical-websites/www.ubuntu.com
# 1603 containers kubernetes

## Done

* update /container/kuberneties to vbt
* added .vimeo-container

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/containers/kubernetes)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/4.%20Containers?preview=containers-kubernetes.png) and [demo](http://www.ubuntu.com-1603-containers-kubernetes.demo.haus/containers/kubernetes)



## Issue / Card

Fixes #1603
